serverless@1.5.0 preuninstall Error: Cannot find module 'bluebird'

# This is a Bug Report

## Description

* What went wrong?

An automatic invocation of `npm install -g serverless` on a macbook that already has serverless installed globally raised a spurious and confusing error message

* What did you expect should have happened?

Nothing, since serverless is already installed globally

* What was the config you used?

Didn't get as far as running serverless before encountering error

* What stacktrace or error message from your provider did you see?

Here is the Terminal log of a demonstration after we figured out what was going on:

```
GSU-ssterling3:~ ssterling3$ npm install -g serverless

> serverless@1.5.0 preuninstall /Users/ssterling3/.npm-global/lib/node_modules/serverless
> node ./scripts/preuninstall.js

module.js:474
    throw err;
    ^

Error: Cannot find module 'bluebird'
    at Function.Module._resolveFilename (module.js:472:15)
    at Function.Module._load (module.js:420:25)
    at Module.require (module.js:500:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/ssterling3/.npm-global/lib/node_modules/serverless/scripts/preuninstall.js:1:81)
    at Module._compile (module.js:573:32)
    at Object.Module._extensions..js (module.js:582:10)
    at Module.load (module.js:490:32)
    at tryModuleLoad (module.js:449:12)
    at Function.Module._load (module.js:441:3)
npm WARN lifecycle serverless@1.5.0~preuninstall: continuing anyway serverless@1.5.0 preuninstall: `node ./scripts/preuninstall.js`
npm WARN lifecycle Exit status 1
/Users/ssterling3/.npm-global/bin/serverless -> /Users/ssterling3/.npm-global/lib/node_modules/serverless/bin/serverless
/Users/ssterling3/.npm-global/bin/slss -> /Users/ssterling3/.npm-global/lib/node_modules/serverless/bin/serverless
/Users/ssterling3/.npm-global/bin/sls -> /Users/ssterling3/.npm-global/lib/node_modules/serverless/bin/serverless

> serverless@1.5.0 postinstall /Users/ssterling3/.npm-global/lib/node_modules/serverless
> node ./scripts/postinstall.js

/Users/ssterling3/.npm-global/lib
└── serverless@1.5.0 

GSU-ssterling3:~ ssterling3$ npm --version
4.0.2
GSU-ssterling3:~ ssterling3$ which npm
/opt/local/bin/npm
GSU-ssterling3:~ ssterling3$ port provides `which npm`
/opt/local/bin/npm is provided by: npm4
GSU-ssterling3:~ ssterling3$ node --version
v7.0.0
GSU-ssterling3:~ ssterling3$ which node
/opt/local/bin/node
GSU-ssterling3:~ ssterling3$ port provides `which node`
/opt/local/bin/node is provided by: nodejs7
GSU-ssterling3:~ ssterling3$ sw_vers
ProductName:	Mac OS X
ProductVersion:	10.12.2
BuildVersion:	16C67
GSU-ssterling3:~ ssterling3$ 
```

Similar or dependent issues:
* (none found)

## Additional Data

* ***Serverless Framework Version you're using***: 1.5.0

* ***Operating System***: macOS 10.12.2 (Sierra)

* ***Stack Trace***: 

```
module.js:474
    throw err;
    ^

Error: Cannot find module 'bluebird'
    at Function.Module._resolveFilename (module.js:472:15)
    at Function.Module._load (module.js:420:25)
    at Module.require (module.js:500:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/ssterling3/.npm-global/lib/node_modules/serverless/scripts/preuninstall.js:1:81)
    at Module._compile (module.js:573:32)
    at Object.Module._extensions..js (module.js:582:10)
    at Module.load (module.js:490:32)
    at tryModuleLoad (module.js:449:12)
    at Function.Module._load (module.js:441:3)
```

* ***Provider Error messages***:

All errors appear to originate in serverless preuninstall script

--- a/scripts/postinstall.js
+++ b/scripts/postinstall.js
@@ -1,10 +1,9 @@
-const BbPromise = require('bluebird');
 const Serverless = require('../lib/Serverless');
 
 const serverless = new Serverless();
 
 (() => {
   serverless.init().then(() => {
-    serverless.utils.logStat(serverless, 'install').catch(() => BbPromise.resolve());
+    serverless.utils.logStat(serverless, 'install').catch(() => Promise.resolve());
   });
 })();

--- a/scripts/preuninstall.js
+++ b/scripts/preuninstall.js
@@ -1,10 +1,9 @@
-const BbPromise = require('bluebird');
 const Serverless = require('../lib/Serverless');
 
 const serverless = new Serverless();
 
 (() => {
   serverless.init().then(() => {
-    serverless.utils.logStat(serverless, 'uninstall').catch(() => BbPromise.resolve());
+    serverless.utils.logStat(serverless, 'uninstall').catch(() => Promise.resolve());
   });
 })();

